Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
Okay. I will stick to something "- 2048"-like then.

IOS is doing the decryption. It would be nice to be able to put Dolphin's decryption in WII_IPC_HLE_Device_DI, but it won't work with the way DiscIO is set up to do it.